bundles/org.openhab.binding.lutron/README.md

```diff
@@ -185,11 +195,14 @@ The correct channels will then be automatically configured.
 
 To support this keypad's contact closure inputs, CCI channels named *cci1* and *cci2* are created with item type Contact and category Switch.
 They are marked as Advanced, so they will not be automatically linked to items in the Paper UI's Simple Mode.
-They accept ON/OFF commands and present ON/OFF states the same as a keypad button.
+They present ON/OFF states the same as a keypad button.
```

Comment:
This is a difference. Was the "accept ON/OFF" part wrong?

Reply:
Yes. That's fixing my own error from a previous commit.